CONTRIBUTING.md

```diff
@@ -1,102 +1,181 @@
+- What did you expect to see?
+- What did you see instead?
 
-  - The changes/feature are reviewed and tested by you
-  - You think it's production ready
-  - The code is linted and the test suite is passing
+### Feature Requests or Enhancements
 
-- It's fine to open a pull request to start a discussion / ask for help, but it should be stated clearly that it's not yet ready for merge.
-- When the `next`-branch is thoroughly tested and ready to be released (either as a minor or major version bump), it should be merged into master and bumped.
-- Critical fixes go straight into master (preferably through a pull request)
+Have you found yourself wishing for a feature that doesn't exist in Sanity? It's likely you are not alone in wanting a particular feature and you should submit a Feature Request. We appreciate and consider all feature requests, and many of the features that Sanity has today have been added because someone saw the need for them.
 
-## Merging
+To submit a Feature Request, please make sure that a request like your own doesn't already exist by searching through our [open issues](https://github.com/sanity-io/sanity/issues), before you create a new one. If someone already submitted the same feature requests you wanted to submit, please let us know you also want this feature by reacting with/adding a :+1: on the original comment, or leave a new comment expressing your need/excitement for this feature.
 
-Prefer squash + merge. If it makes sense to keep individual commits (e.g. different people has been working on the same feature), the pull request should be merged with the "Rebase + merge"-button. If possible, each individual commit message should be rewritten with the pull-request number in parenthesis, e.g. `[some-package] Fix stuff (#22)`
+If the same Feature Request doesn't already exist, create a new issue and use the Feature Requests template. Be sure to provide all the necessary information about your new feature to help us understand what you are requesting.
 
-## Publishing official releases
+### Pull Requests
 
-[**NOTE** Only publish _minor_ releases until Sanity passes the 1.0 mark]
+When you are done working on a fix or improvement to the Sanity Studio, make sure your branch is even with the `next` branch of the original Sanity repository by rebasing it against `next`.
 
-When `next` is ready for release, run `npm run publish` and select version. After a successful release, remember to rebase `next` against `master`.
+```
+git pull upstream next --rebase
+```
+
+Once this is done and there are no conflicts, push your changes and create a Pull Request from your branch and provide the necessary information for your fix or improvement. Be sure to follow the Pull Request template to the best of your ability, to help us easily review it and provide feedback faster.
+
+# Setting up Sanity locally
 
-## Publishing a hotfix release from `master`
+Make sure you have _Node.js version 4 or newer_ and _npm version 5 or newer_.
```

Comment:
I highly doubt this works in Node 4. Don't we require at least Node 8?

Reply:
Should be node 10 now